decidim.gemspec

```diff
@@ -39,6 +39,7 @@ Gem::Specification.new do |s|
   s.add_dependency "decidim-assemblies", Decidim.version
   s.add_dependency "decidim-budgets", Decidim.version
   s.add_dependency "decidim-comments", Decidim.version
+  s.add_dependency "decidim-consultations", Decidim.version
```

Comment:
I'm thinking that maybe we could leave consultations (and potentially other gems) outside of the core bundle. They could leave inside the decidim repo (and test against it) but be optional. It's a very specific feature that most users won't use.

What do you think? @andreslucena @oriolgual @mrcasals ?

Reply:
I think that's confusing for the end developer

Reply:
I'd keep it too, although maybe @decidim/product should choose, since it's not a development issue